README.md

```diff
@@ -2,272 +2,36 @@
 
 ## Configuration
 
-Check out
-[`web/modules/custom/hoeringsportal_deskpro/README.md`](web/modules/custom/hoeringsportal_deskpro/README.md)
-for details on configuring the Deskpro integration.
+### Guide for local development README
```

Comment:
Remove “README” from the headings. Maybe the headings themselves can be removed and the links added in a list (like a table of contents).

Reply:
Removed and changed